scripts/group-names.ts

```diff
@@ -104,27 +105,18 @@ async function main() {
         default:
           throw new Error(`Unexpected case '${name.case}'`);
       }
-      byCategory[name.category] ||= {};
-      byCategory[name.category][name.gender] ||= {};
-      byCategory[name.category][name.gender][_case] ||= name;
+      for (const category of name.categories) {
+        byCategory[category] ||= {};
+        byCategory[category][name.gender] ||= {};
+        byCategory[category][name.gender][_case] ||= name;
+      }
     }
 
     let category: string | undefined;
     const categories = Object.keys(byCategory);
 
     assert(categories.length > 0, "should have at least 1 category");
 
-    if (
-      categories.length === 1 &&
-      ["gæl,ism", "dýr,hetja"].includes(categories[0])
-    ) {
-      // This seems like a data entry error, for which BÍN should be contacted.
-      // These occur for 1 word each.
-      //
-      // Anyway, ignore these while they are sorted out in the source.
-      continue;
-    }
```

Comment:
A new entry with multiple categories has appeared in the source data. "Atlas" contains a single entry with the categories `hetja,ism`.

Given that BÍN is adding new entries with multiple word categories, I've updated `beygla` to handle multiple word categories in a single entry.

However, it is weird that the source data is inconsistent about this.